### PR TITLE
Merge docker efi images.

### PR DESCRIPTION
**Pull request by Hnefi**
IDK why this isn't automatically merged?

Maybe because I did ```--tree-filter ``` many times to rewrite the history.

### CODE REVIEW

**Comment by neo-apz** [comment]
There is a conflict in `.gitattributes` and it's because your branch is 10 commits behind master.
You need to pull (rebase) the new commits from master into your branch and update your branch.

**Comment by Hnefi** [comment]
@neo-apz 
All commits rebased, please verify correctness and merge at your leisure.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+new-ubuntu/ubuntu-16.04.3-efi-docker.qcow2
```
Comment (neo-apz) [inline .gitignore:1]: Why is this needed?

File: ubuntu-16.04-blank/.gitattributes [hunk 2]
```diff
@@ -0,0 +1 @@
+ubuntu-16.04-lts-blank.qcow2.bz2 filter=lfs diff=lfs merge=lfs -text
```
Comment (neo-apz) [inline ubuntu-16.04-blank/.gitattributes]: Wouldn't it be better to have all of the LFS-related lines in one file (`.gitattributes` in the root)?

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule "edk2"]
```
Comment (neo-apz) [inline .gitmodules]: I have some doubts regarding this being a submodule.
If it's something needed just for the build phase, it can be installed then based on the wiki pages mentioned. Why do we need to keep its source in our repo?

**Review by neo-apz — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Hnefi) [reply .gitignore:1]: Because I don't want it trying to commit the 6GB image accidentally. The compressed one is stored in git-lfs, this one I want to forever exclude from being tracked.

File: .gitmodules (hunk 3 above)
Comment (Hnefi) [reply .gitmodules]: I wanted it to be continuously updated as we are doing with QEMU, since it is still under active development (mainly driven by requirements for KVM).

File: ubuntu-16.04-blank/.gitattributes (hunk 2 above)
Comment (Hnefi) [reply ubuntu-16.04-blank/.gitattributes]: Yes, I have added another commit to fix this.

File: .gitignore (hunk 1 above)
Comment (neo-apz) [reply .gitignore:1]: 👍

**Review by neo-apz — APPROVED** [review]
